### ISSUE TITLE
Emoji are small again

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#22086]
### Steps to reproduce

Send or receive a message with an emoji, or insert one into the composer

#### What did you expect?

Emoji should be larger than the surrounding text

#### What happened instead?

They aren't

### Operating system

NixOS unstable

### Browser information

Firefox 99.0.1

### URL for webapp

develop.element.io

### Application version

Element version: abb31c9b01a2-react-b5ac9493dd14-js-6b5f4aa0a9c1 Olm version: 3.2.8

### Homeserver

Synapse 1.58.0

### Will you send logs?

No
